schema/filter_artworks.js

```diff
@@ -132,13 +133,22 @@ function filterArtworks(primaryKey) {
       },
     },
     resolve: (root, options, request, { rootValue: { accessToken } }) => {
+      const gravityOptions = options;
```

Comment:
Might want to make this a copy as to not mutate the original? `Object.assign({}, options)` iirc